src/frontend/components/VideoWizard/CreateVOD/index.tsx

```diff
@@ -0,0 +1,210 @@
+  placeholderTitleInput: {
+    defaultMessage: 'Enter title of your video here',
+    description:
+      'A placeholder text indicating the purpose of the input and what it is supposed to received.',
+    id: 'components.CreateVOD.placeholderTitleInput',
+  },
+  goBackButtonLabel: {
+    defaultMessage: 'Go back',
+    description:
+      "Button's label offering the user to go back to the last screen.",
+    id: 'components.CreateVOD.goBackButtonLabel',
+  },
+  createVideoButtonLabel: {
+    defaultMessage: 'Create a video',
+    description:
+      "Button's label offering the user to go validate is form and create a new video.",
+    id: 'components.CreateVOD.createVideoButtonLabel',
+  },
+  updateVideoInfosFail: {
+    defaultMessage: 'Video infos update has failed !',
+    description: 'Message displayed when video infos update has failed.',
+    id: 'component.CreateVOD.updateVideoInfosFail',
+  },
+});
+
+interface WizardedVideo {
+  title: Video['title'];
+  videoFile: Nullable<File>;
+  license: Video['license'];
+}
+
+interface CreateVODProps {
+  video: Video;
+}
+
+const CreateVOD = ({ video }: CreateVODProps) => {
+  const history = useHistory();
+  const currentVideo = useVideo((state) => state.getVideo(video));
+  const intl = useIntl();
+  const size = useContext(ResponsiveContext);
+  const [wizardedVideo, setWizardedVideo] = useState<WizardedVideo>({
+    title: null,
+    videoFile: null,
+    license: null,
+  });
+  const [isVideoValidated, setIsVideoValidated] = useState(false);
+
+  const { addUpload, uploadManagerState } = useUploadManager();
+
+  const videoMutation = useUpdateVideo(currentVideo.id, {
+    onError: (err) => {
+      report(err);
+      toast.error(intl.formatMessage(messages.updateVideoInfosFail), {
+        position: 'top-center',
+      });
+    },
+  });
+
+  const canValid =
+    wizardedVideo.title &&
+    wizardedVideo.title !== '' &&
+    wizardedVideo.videoFile &&
+    wizardedVideo.license &&
+    wizardedVideo.license !== 'error';
+
+  useEffect(() => {
+    if (
+      uploadManagerState[currentVideo.id] &&
+      uploadManagerState[currentVideo.id].progress === 100 &&
+      currentVideo.upload_state === uploadState.PROCESSING
+    ) {
+      videoMutation.mutate({
+        title: wizardedVideo.title,
+        license: wizardedVideo.license,
+      });
+      // If title & license update fails, we still redirect the user to the dashboard
+      // because it will know update has failed (react toast) but he will be able
+      // to update it using the dashboard
+      history.push(DASHBOARD_ROUTE(modelName.VIDEOS));
+    }
+  }, [uploadManagerState[currentVideo.id], currentVideo]);
+
+  return (
+    <CurrentVideoProvider value={currentVideo}>
+      <WhiteCard title={intl.formatMessage(messages.videoCreationTitle)}>
+        <Box
+          direction="column"
+          gap="medium"
+          margin={{ horizontal: size === 'medium' ? 'medium' : 'xlarge' }}
+        >
+          <Text
```

Comment:
The title chosen here is replaced later by the video file name. Once the video uploaded and redirected to the dashboard, we see the video file name and not the title from this form.